apps/www/next.config.js

```diff
@@ -55,11 +55,29 @@ module.exports = withTM(
     },
     async redirects() {
       return [
+        // '/front' must not be reachable directly!
+        // if a user is authenticated, '/' will be rewritten to '/front'
+        {
+          source: '/front',
+          destination: '/',
+          permanent: true,
+        },
```

Comment:
`permanent`  might not be ideal since we lock that path if we'd ever want to use it
```suggestion
        {
          source: '/front',
          destination: '/',
        },
```
or even rewrite to 404